org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -4743,25 +5046,20 @@ class ExecuteCommandParams {
 		this.command = Preconditions.checkNotNull(command, 'command')
 		this.arguments = arguments
 	}
+	
+	new(@NonNull String command, List<Object> arguments, ProgressToken workDoneToken) {
+		this.workDoneToken = workDoneToken
```

Comment:
Nit: For consistency please assign the fields in the same order as the parameter lists.